### ISSUE TITLE
Exception when adding primary ip to device

### ISSUE THREAD

**Issue by p-rintz** [nautobot/nautobot#300]
### Environment
* Python version:3.8.5
* Nautobot version: 1.0.0b3

### Steps to Reproduce
1. Go to any device
2. Set a primary ip (either v4 or v6)
3. Click on update

### Expected Behavior
That it succeeds in setting the primary ip.

### Observed Behavior
It will throw an exception in the [deconstruct_address](https://github.com/nautobot/nautobot/blob/b2135c0d40632b03a5a90fe948030c3bb0009020/nautobot/ipam/models.py#L825) function, as the IPAddress Class does not have an IP variable.
Additonally, when this is fixed, the prefixlen in the same function will also throw an exception as it seems to have been renamed prefix_length in the class in the following [commit](https://github.com/nautobot/nautobot/commit/c937b65830c12667b031cfa17047f5f7db91bc52#diff-15c6afc90d7ab9ffc408a2b7c6857caf4c3ebb9bc1213aa5340cbc301f85e7c0R742).

It generally seems to me as if there are a few parts in the model that still reference old variables.

**Comment by glennmatthews** [comment]
Thanks for the report! As discussed on Slack, I'm able to confirm this issue locally as well - we'll get it fixed!

**Comment by tim-fiola** [comment]
In beta4 I confirmed this in the UI, but I do not see the problem when adding/changing the device primary IP via rest calls.

Did a PATCH: `127.0.0.1:8000/api/dcim/devices/6439f6df-5d57-4ccc-aee2-c7d4ddfb4a4d/`

with body: 
```
{
"primary_ip4": "9b3cfc16-02da-467d-9d03-228b4915ce10"
}
```


Adding the postman collection where I validated the rest testing:
```
{
	"info": {
		"_postman_id": "5ee545fe-b275-46f1-b67f-9548a26fc786",
		"name": "z-local testing",
		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
	},
	"item": [
		{
			"name": "get device",
			"request": {
				"method": "GET",
				"header": [],
				"url": {
					"raw": "127.0.0.1:8000/api/dcim/devices/6439f6df-5d57-4ccc-aee2-c7d4ddfb4a4d/",
					"host": [
						"127",
						"0",
						"0",
						"1"
					],
					"port": "8000",
					"path": [
						"api",
						"dcim",
						"devices",
						"6439f6df-5d57-4ccc-aee2-c7d4ddfb4a4d",
						""
					]
				}
			},
			"response": []
		},
		{
			"name": "assign primary ip to ams-edge-02",
			"request": {
				"method": "PATCH",
				"header": [],
				"body": {
					"mode": "raw",
					"raw": "{\n\"primary_ip4\": \"90fb6f87-4c6c-45dd-b4ed-5c7d5d73d2bb\"\n}",
					"options": {
						"raw": {
							"language": "json"
						}
					}
				},
				"url": {
					"raw": "127.0.0.1:8000/api/dcim/devices/6439f6df-5d57-4ccc-aee2-c7d4ddfb4a4d/",
					"host": [
						"127",
						"0",
						"0",
						"1"
					],
					"port": "8000",
					"path": [
						"api",
						"dcim",
						"devices",
						"6439f6df-5d57-4ccc-aee2-c7d4ddfb4a4d",
						""
					]
				}
			},
			"response": []
		}
	],
	"auth": {
		"type": "apikey",
		"apikey": [
			{
				"key": "value",
				"value": "Token 671fd97309546334b917a27f196b04ff4d4ba9bc",
				"type": "string"
			},
			{
				"key": "key",
				"value": "Authorization",
				"type": "string"
			}
		]
	},
	"event": [
		{
			"listen": "prerequest",
			"script": {
				"type": "text/javascript",
				"exec": [
					""
				]
			}
		},
		{
			"listen": "test",
			"script": {
				"type": "text/javascript",
				"exec": [
					""
				]
			}
		}
	]
}
```

**Comment by glennmatthews** [comment]
Fixed by #301.
